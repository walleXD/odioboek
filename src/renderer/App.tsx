import React from 'react'
import { HashRouter } from 'react-router-dom'

import Pages from './pages'

const App = () => (
    <div>
        <HashRouter>
            <Pages />
        </HashRouter>
    </div>
)

export default App