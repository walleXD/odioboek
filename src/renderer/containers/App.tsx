import React from "react"
import { HashRouter } from "react-router-dom"
import Container from "@material-ui/core/Container"
import Box from "@material-ui/core/Box"
import { Helmet } from "react-helmet"
import Pages from "../pages"

const App: React.FunctionComponent = (): React.ReactElement => (
  <Container>
    <Box>
      <Helmet defaultTitle="Odioboek" titleTemplate="%s" />
      <HashRouter>
        <Pages />
      </HashRouter>
    </Box>
  </Container>
)

export default App
