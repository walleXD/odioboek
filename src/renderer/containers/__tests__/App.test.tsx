import "jest-dom/extend-expect"
import * as React from "react"
import { render, cleanup } from "react-testing-library"

import App from "../App"

describe("App Component", (): void => {
  afterEach(cleanup)

  test("renders without crashing", (): void => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
})
