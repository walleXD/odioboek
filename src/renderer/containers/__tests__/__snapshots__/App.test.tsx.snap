// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      class="MuiBox-root MuiBox-root-9"
    >
      <p
        class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
      >
        Odioboek
      </p>
    </div>
  </div>
</DocumentFragment>
`;
